server.go

```diff
@@ -797,6 +796,13 @@ func (s *Server) Users() (a []*User) {
 	return a
 }
 
+// UsersLen returns the number of users.
+func (s *Server) UsersLen() int {
```

Comment:
I created this function because otherwise we'd be creating a new array, filling it with all users, and then sorting it.  All that twice for every query that came in.

I started with `UsersCount` and flip flopped.  I'll change it back.  Maybe `UserCount` would be even better?